### PR TITLE
Am scss variables

### LINKED ISSUE #40: SCSS: variables.scss
Refactor CSS variables where possible

### LINKED ISSUE #42: SCSS: color functions
Use color function where possible

### PR DESCRIPTION
**Pull request by aemcdonald**
### What’s this PR do?  
Fixed all images
Created different scss files for related features
Added all necessary imports to styles.scss
Nested like-features in appropriate styles
Created variables for repeating colors in scss files
Renamed all instances of repeating colors with appropriate variable names

### Where should the reviewer start?  
Index.js
Scripts.js
All scss files

### How should this be manually tested?  
`npm run build` may need to be run after pulling changes down to configure webpack on your local machine. 

### Any background context you want to provide?  
Ran `npm run build` to configure webpack files in order to fix all images. Removed unnecessary scripts tag due to webpack configuration.

### What are the relevant tickets?  
Closes #42 
Closes #40

### CODE REVIEW

**Review by npdarrington — APPROVED** [review]
Very nice splitting everything up and making use for scss variables to compress these scss files. It will be much easier to debug if an issue arises with a style in this new format! Also a good place for a PR before getting really deep in compressing like-minded styles.

File: src/css/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+$lemon-lime: #C4EB67;
```
Comment (JWoltjen) [inline src/css/_variables.scss:1]: Nice color naming schemes

**Review by JWoltjen — APPROVED** [review]
Nice integration of webpack with the rest of the files, as well as using SCSS to replace css styles that are used more than once.

File: src/css/_buttons.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.show-pantry-recipes-btn,
```
Comment (hannakim91) [inline src/css/_buttons.scss:1]: These files make it much easier to read the CSS

File: src/css/_header.scss [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  font-family: "Quicksand", sans-serif;
+  font-size: 42px;
+  height: 100%;
+  outline: none;
+  padding: 0% 2% 0% 0%;
+  text-align: right;
+  transition: .8s ease;
+  width: 100%;
+
+  &:hover {
```
Comment (hannakim91) [inline src/css/_header.scss:48]: This looks like a good use case for nesting!

File: src/css/_variables.scss (hunk 1 above)
Comment (hannakim91) [inline src/css/_variables.scss:1]: Fun variable naming!

File: src/index.js [hunk 4]
```diff
@@ -1,13 +1,9 @@
...
-// Do not delete or rename this file ********
-// An example of how you import jQuery into a JS file if you use jQuery in that file
-import $ from 'jquery';
-
-// An example of how you tell webpack to use a CSS (SCSS) file
-import './css/base.scss';
-
-// An example of how you tell webpack to use an image (also need to link to it in the index.html)
+//Image Import
+import './images/apple-logo-outline.png'
```
Comment (hannakim91) [inline src/index.js:2]: Glad we finally figured out how to get the broken images to show!

**Review by hannakim91 — APPROVED** [review]

**Pull request merged**